apis/v1alpha1/generator.yaml

```diff
@@ -1,5 +1,65 @@
 ignore:
   resource_names:
-  - Pipe
+      # - Pipe
 sdk_names:
-  model: pipes
\ No newline at end of file
+  client_struct: Pipes
+  client_interface: PipesAPI
+resources:
+  Pipe:
+    fields:
```

Comment:
Let me know if we need custom compare for tags as per API def they're a `map` type (which I guess you'd consider non-deterministic) -> [Tags](https://docs.aws.amazon.com/eventbridge/latest/pipes-reference/API_DescribePipe.html#eventbridge-DescribePipe-response-Tags).

Reply:
Just added one :)